frontend/src/App.tsx

```diff
@@ -16,6 +16,7 @@ export default class App extends Component<Props, State> {
     render() {
         return (
             <Switch>
+                <Route path="/components" component={ComponentsView} />
```

Comment:
Dałoby się to dodać tylko w developmencie? Znalazłem coś takiego: https://stackoverflow.com/questions/35469836/detecting-production-vs-development-react-at-runtime

Ale może react ma jakieś fancy biblioteki do tego :)

Reply:
Zrobię na to ticketa, zrobię to jakoś specjalny routing